doc/Standaard-inrichting-GitHub-GitLab.md

```diff
@@ -16,38 +16,45 @@ Een aanvraag voor een nieuwe VNG Realisatie GitHub repository wordt altijd per m
+4. Stel de repository als 'Public' in wanneer deze public moet zijn;
+5. Wanneer het een nieuwe repository betreft en er nog geen inhoud is: selecteer de optie 'Add a README file'. Er wordt dan een standaard README.md bestand aangemaakt waarvan de content later kan worden aangebracht;
+Bij VNG Realisatie nemen wij altijd zo'n bestand op bij een lege repository; 
+Bestaat er al een repository die moet worden overgezet maak je geen README aan. Deze bestaande repository wordt dan geïmporteerd in de nieuwe.
 6. We creëren niet standaard een licentie bestand en ook geen .gitignore;
 7. Klik op 'Create repository'.
 
 ### Gebruikers (autorisaties)
 
-Doordat de repository 'Public' is, staat deze voor de gehele wereld open. Dat betekent dat de inhoud door de gehele wereld bekeken kan worden. Dat is ook
-precies zoals wij dat bij VNG Realisatie willen. 
+Wanneer de repository 'Public' is, staat deze voor de gehele wereld open. Dat betekent dat de inhoud door de gehele wereld bekeken kan worden. Het is dan niet nodig om gebruikers expliciet leesrechten te geven, die heeft immers iedereen al.
+Er zijn ook gevallen waarbij informatie juist niet voor de hele wereld open moet staan, bijvoorbeeld omdat deze misbruikt zou kunnen worden voor oneigenlijke doeleinden. In die gevallen maken we een private repository. In zo'n geval moeten gebruikers wel expliciet leesrechten krijgen om de inhoud te kunnen lezen.
+
 
 1. Ga naar het tabblad 'Settings' en klik op 'Manage access';
 2. Voeg nu m.b.v. de bullet 'Invite teams or people' gebruikers toe. Je kunt hier dus kiezen voor individuele gebruikers of voor reeds aangemaakte teams. 
 Indien je dat wil kan je er ook voor kiezen om eerst een nieuw team aan te maken;
 3. Indien je op 'Invite teams or people' hebt geklikt kom je nu in een interface waarin je een gebruikersnaam of teamnaam in kunt geven;
 4. Definiëer vervolgens de rol die de gebruiker of het team dient te krijgen (Read, Triage, Write of Maintain);
-5. Klik vervolgens op de groene button 'Add xxxxx to the repository' waarna de gebruiker of team gebruik kan maken van de hem/hun toegekende rechten op de
+6. Klik vervolgens op de groene button 'Add xxxxx to the repository' waarna de gebruiker of team gebruik kan maken van de hem/hun toegekende rechten op de
 repository.
+7. De beheerders van de organisatie VNG Realisatie zijn altijd administrator bij elke repository. Je kunt dit eventueel transparant maken voor alle gebruikers door hen ook expliciet die rechten toe te kennen. 
+
+VRAAG/OPMERKING: Het project is (mede)eigenaar van de repository. Binnen een project is iemand verantwoordelijk voor het beheer van de repository. Ook om te voorkomen dat de beheerders van de VNG github omgeving alle projectne in de gaten moeten houden om te zien wie wel/niet meer toegang zou moeten hebben etc. Het project weet dat zelf veel beter. Er is dus altijd iemand vanuit het project als maintainer verantwoordelijk voor het bijhouden van "de administratie".
```

Comment:
De tweede en laatste zin in deze alinea zijn dubbelop. Maak hiervan:

> Een repository wordt vervaardigd i.h.k.v. een project. Ter voorkoming dat de beheerders van de VNG Realisatie github omgeving alle projecten in de gaten moeten houden wordt er altijd iemand van het projectteam als maintainer verantwoordelijk gemaakt voor het bijhouden van "de administratie". Het projectteam weet dat immers zelf het beste wie wel/niet meer toegang zou moeten hebben etc...

Reply:
verwerkt

Reply:
De laatste zin luidt nu:

> Het projectteam weet dat immers zelf het beste wie wel/niet meer toegang zou moeten hebben

Corrigeer dat nog even naar:

> Het projectteam weet immers zelf het beste wie wel/niet meer toegang zou moeten hebben etc...